apps/widget-e2e/src/util/ScatterChartErrorBars.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Chart, IChartElement } from "./Chart";
+
+const dReg =
```

Comment:
interesting, just for my understanding, how does this regular expression work?

Reply:
d attribute is something like below:
 M1182.6,272.53h8m-4,0V342.5m-4,0h8.
This format is from svg grammar ([link](https://www.w3.org/TR/SVG/paths.html)) for drawing graphs. We extract numbers from here and use x/y/heights etc to verify the graph is generated as expected.